docs/knowledge_base/threat_feed.md

```diff
@@ -97,3 +99,42 @@ For example, if you want to limit the items per page to 3 since July 19, 2023 yo
 ```text
 https://threats.phylum.io/?per_page=3&since=2023-07-19
 ```
+
+## Cursor Based Pagination
+The threat feed supports cursor based pagination. By providing the `cursor` parameter you can iterate through the data in a guaranteed order. For example:
+
+```bash
+curl "https://threats.phylum.io/?cursor=84&per_page=10" -H "Authorization: Bearer $PHYLUM_API"
+```
+
+If additional rows exist, the response will include the next cursor in the `cursor` key:
```

Comment:
What about when no additional rows exist? A review of the code that generates and returns the cursor makes me think the `cursor` for the last "page" of results will point to a package that was already displayed (the last one on that last page).

Reply:
After further testing, it appears that the last "page" of results will not include the very last package, even if there is enough `per_page` space to do so. That means the penultimate page of results will have a cursor that points to a single package. Returning that page will then show the cursor value of `null`.

Reply:
You're right. I'll get a fix out for this specifically.

Reply:
https://github.com/phylum-dev/threat-feed/pull/187

Reply:
The code change in phylum-dev/threat-feed#187 appears to correct the issue. The original comment here still stands...what about when no additional rows exist? That is, I think it might be more clear to add another sentence here that makes it explicit. Something like:

> The cursor value will be `null` when no additional rows exist.